go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/hashicorp/levant
 go 1.13
 
 require (
-	github.com/Masterminds/sprig/v3 v3.1.0
+	github.com/Masterminds/sprig/v3 v3.2.1
```

Comment:
Can do. I originally made this change because the 3.1.0 version was not working correctly with `parseJSON` and sprig's `mergeOverwrite`, like from the [learn materials here](https://learn.hashicorp.com/tutorials/nomad/levant-abstract-jobs#allow-the-defaults-to-be-overridden). I forget what the exact error was, but it was something to do with merging `interface{}` with `map[string]interface{}` and how they are not compatible.